### DISCUSSION TITLE
Request: Ability to change the number of connections for downloads

### DISCUSSION THREAD

**Discussion by PCUser2021** [setvisible/DownZemAll#187] [Ideas]
Just like all download managers.
By the way, thanks for resurrecting the DTA!

**Comment by setvisible** [comment]
Hi

Did you try:

![1](https://user-images.githubusercontent.com/20563751/166226688-be662894-4e78-41cf-94e9-5b5dbe65240b.png)


(DTA still exists I think... on the form of a web extension)
